src/static/styles/App.scss

```diff
@@ -675,3 +751,76 @@ div.scrollbar {
 .statement-solidarity {
 	background-color: $secondary-theme-light;
 }
+
+.search-bar-div {
+	margin-left: 8rem;
+	display: flex;
+	justify-content: left;
+}
+
+.preview-button {
+	color: rgb(68, 95, 235);
+	font-weight: bolder;
+}
+
+.double-button-jawn {
+	display: flex;
+	text-align: center;
+	justify-content: center;
+}
+
+.subheader {
+	color: #6C757D;
+}
+
+input#search {
+	background-image: url('https://cdn0.iconfinder.com/data/icons/very-basic-2-android-l-lollipop-icon-pack/24/search-512.png');
+	background-size: 1.6rem;
+	background-position-y: center;
+	background-repeat: no-repeat;
+	text-indent: 1rem;
+	transition: 0.3s;
+}
+
+input#search:focus {
+	background-image: none;
+	text-indent: 0
+}
+
+.modal-content {
+	background-color: transparent !important;
+	border: 0 !important;
+}
+  
+.modal-header {
+	border: 0 !important;
+}
+
+.modal {
+	background-color: rgba(0, 0, 0, 0.625);
+}
+
+.unfilled-text {
+	color:rgb(228, 13, 13)
+}
+
+.transparent-button {
+	background: transparent;
+	border: none;
+	margin-left: auto;
+	margin-right: 0;
+}
+
+input#search-bar-0.form-control {
+	background-image: url('https://cdn0.iconfinder.com/data/icons/very-basic-2-android-l-lollipop-icon-pack/24/search-512.png');
```

Comment:
are we hardcoding images here? whats the difference between this and the images you added in the static folder?